content/docs/user-guide/basic-concepts.md

```diff
@@ -0,0 +1,129 @@
+# Basic Concepts
+
+Intro and DVC philosophy...possible diagram of cache/remote/workspace
+
+## Cache
+
+_From `dvc cache`_
+
+The DVC Cache is where your data files, models, etc. (anything you want to
+version with DVC) are actually stored. The data files and directories visible in
+the <abbr>workspace</abbr> are links\* to (or copies of) the ones in cache.
+Learn more about it's
+[structure](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory).
+
+_from tooltip_
+
+The DVC cache is a hidden storage (by default located in the `.dvc/cache`
+directory) for files that are tracked by DVC, and their different versions.
+Learn more about it's
+[structure](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory).
```

Comment:
OK I take it it's an intermediate step to put this here right? Because we don't want to duplicate these texts exactly, as the tooltips will (most likely) also remain.

Reply:
> OK I take it it's an intermediate step to put this here right?

Yes, I placed it as a mock-up for discussion purposes.

Reply:
Cool